go/vm/lfvm/instructions_test.go

```diff
@@ -465,3 +466,116 @@ func TestMCopy(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateShanghaiInitCodeSize(t *testing.T) {
+	maxInitCodeSize := uint64(49152)
+	tests := []struct {
+		revision       vm.Revision
+		init_code_size uint64
+		expected       Status
+	}{
+		{vm.R11_Paris, 0, RUNNING},
+		{vm.R11_Paris, 1, RUNNING},
+		{vm.R11_Paris, 2000, RUNNING},
+		{vm.R11_Paris, maxInitCodeSize - 1, RUNNING},
+		{vm.R11_Paris, maxInitCodeSize, RUNNING},
+		{vm.R11_Paris, maxInitCodeSize + 1, RUNNING},
+		{vm.R11_Paris, 100000, RUNNING},
+		{vm.R11_Paris, math.MaxUint64, RUNNING},
```

Comment:
It is surprising that this test is not failing -- either with an out-of-gas error or a bad allocation crash.

I looked into it, and it looks like `CREATE` is not growing the memory to fit its input requirements. Is this something that is missing? If so, why did this not show up in the CT?

Reply:
This should be investigated and solved in #524.
I added this as a comment in the code since it should not be part of this PR.